<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enterCfbDialog</class>
 <widget class="QDialog" name="enterCfbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter CFB IV/Key</string>
  </property>
  <widget class="QLabel" name="IvLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Initialization Vector:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ivLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff / 255, 0x11 / 17...</string>
   </property>
  </widget>
  <widget class="QLabel" name="helpLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Either hexadecimal or decimal numbers, separated by a coma(',')</string>
   </property>
  </widget>
  <widget class="QLabel" name="frameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter frame:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="framLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0x21</string>
   </property>
  </widget>
  <widget class="QLabel" name="keysLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter keys: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0x00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xfc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyLineEdit_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g. 0xff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
